app/javascript/react/components/molecules/Calendar/Calendar.style.tsx

```diff
@@ -1,9 +1,25 @@
 import styled from "styled-components";
+import media from "../../../utils/media";
 
 const ThreeMonths = styled.div`
   display: flex;
   justify-content: center;
-  gap: 40px;
+  gap: 2rem;
```

Comment:
In designs it's 30px - 3rem

Reply:
This was my way to gain more "space" for the Day component :) But surely I can change it as it is in design